language/resource.language.en_gb/strings.po

```diff
@@ -276,6 +276,10 @@ msgctxt "#772"
 msgid "Checking for a new version will submit the installed Distribution, Project, CPU Arch and Version to the update server. This data is also used to report basic statistics for the project. To continue checking for updates while opting-out of stats reporting, disable this option."
 msgstr ""
 
+msgctxt "#773"
+msgid "Set the idle timeout (in minutes) before devices will be disconnected (defaults to 0 for no timeout). Only applies to devices with enabled standby."
```

Comment:
```suggestion
msgid "Set the idle timeout (in minutes) before devices will be disconnected (defaults to 0 for no timeout). Only applies to devices with standby support."
```
Is that better?

Reply:
Sounds better, yeah.
But the feature only works on devices where you set "Enable Standby" in the device's context menu. That's a LE feature, not some general standby support of the hardware. Maybe quote "Enable Standby" in this msg?

Reply:
Ah ok, I see. Yes, quoting would help.

Or maybe:
'''
Applies to devices where "Enable Standby" has been activated in the LibreELEC device connection context menu.
```

Reply:
Sold!